codec-http2/src/test/java/io/netty/handler/codec/http2/HpackEncoderTest.java

```diff
@@ -68,7 +68,6 @@ public void execute() throws Throwable {
     /**
      * The encoder should not impose an arbitrary limit on the header size if
      * the server has not specified any limit.
-     * @throws Http2Exception
```

Comment:
But the method throws `Http2Exception`, right?

Reply:
This is a test, not public API.

Reply:
Ah, Alright then.